documentation/docs/getting_started/java/getting_started.mdx

```diff
@@ -72,18 +74,18 @@ Generating the jar can be done with your tool of preference. We provide examples
 Make `gradlew` executable (`chmod +x gradlew`) if needed, then run
 
 ```bash
-cd iota.rs/bindings/java
+cd iota.rs/bindings/java/iota-client-java/
```

Comment:
Thanks for your PR, but this looks like it should target the `develop` branch, no?